samples/bluetooth/broadcast_audio_sink/boards/nrf5340dk_nrf5340_cpuapp.conf

```diff
@@ -0,0 +1,3 @@
+# The LC3 codec uses a large amount of stack. This app runs the codec in the work-queue, hence
+# inctease stack size for that thread.
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=4096
```

Comment:
Hmm, I wonder if this is actually needed. I think it was originally added for LC3 encoding, but the sink only does decoding. 

@larsgk @fredrikdanebjer do you have any experience with the stack size requirements for the decoder?

Reply:
Note this is just copy pasted from the other board .conf that was already there